openapi/components/examples/lambdas/200-third-party-integrations.json

```diff
@@ -0,0 +1,16 @@
+{
+  "data": [
+    {
+      "urn": "urn:decentraland:ethereum:ext:cryptopunks:0xContractAddressaaabbff999123",
```

Comment:
I would use the correct definition of the third party `urn` :
`urn:decentraland:{protocol}:collections-thirdparty:{third-party-name}:{collection-id}:{token-id}`